modules/understanding-upgrade-channels.adoc

```diff
@@ -82,6 +83,27 @@ and pre-production and production environments that participate in connected cus
 You can use the `stable-{product-version}` channel to upgrade from a previous minor version of
 {product-title}.
 
+[discrete]
+== eus-{product-version} channel
+
+In addition to the stable channel, certain minor versions of {product-title} offer an
+Extended Update Support[https://access.redhat.com/support/policy/updates/openshift#ocp4_phases] (EUS).
+These EUS versions extend the maintenance phase for customers with Premium Subscriptions
+to 14 months. {product-title} {product-version} is a minor version that includes EUS.
+
+Although there is no difference between stable-{product-version} and eus-{product-version}
+channels until {product-version} transitions to EUS phase, you can switch
+to the EUS channel as soon as it becomes available. When {product-title} {product-version}
+transitions to the EUS phase of its lifecycle, the stable-{product-version} channel will
+no longer receive updates. After you upgrade a to a version that is exclusive
+to the EUS channel, that cluster is no longer be eligible for minor version upgrades
+until upgrades to the next EUS version become available. Currently, upgrades between
+EUS versions requires a serial set of version upgrades, such as from 4.6 to 4.7 to 4.8 to 4.9 to 4.10.
+
+Additionally, you must upgrade to {product-title} {product-version} before it
+transitions to EUS phase. For example, you cannot upgrade from 4.5 to 4.6 after 4.6 transitions
+to EUS status.
```

Comment:
Is this an important aspect of the EUS phase?  By the time 4.6 hits end-of-maintenance (and the beginning of EUS), 4.5 will have been EOL for a while, so those folks are already out of support.  Say 4.5.99 is the last 4.5.z, and 4.6.90 is the last end-of-maintenance 4.6.z.  Folks who are on 4.5.99 when we enter EUS with 4.6.91 can still go 4.5.99 -> 4.6.90 -> 4.6.91, without going more stale than they were when they were running 4.5.99.  The fact that we are not currently planning to add direct 4.5.99 -> 4.6.91 edges does not seem important enough to be worth calling out in the channel docs.

Reply:
I just simplified it to say that you have to be running a supported version of 4.6 before switching to eus-4.6. PTAL